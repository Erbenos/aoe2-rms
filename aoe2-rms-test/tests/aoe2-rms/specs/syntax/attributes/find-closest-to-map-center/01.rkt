#lang aoe2-rms
; Place a boar in the map center for each player.

<OBJECTS-GENERATION>
(create-object 'BOAR
               (set-place-for-every-player)
               (set-gaia-object-only)
               (find-closest-to-map-center))
