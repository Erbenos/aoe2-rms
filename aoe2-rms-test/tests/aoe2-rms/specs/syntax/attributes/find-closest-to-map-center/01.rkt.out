<OBJECTS_GENERATION>
create_object BOAR {
  set_place_for_every_player
  set_gaia_object_only
  find_closest_to_map_center
}
